railgun/internal/proxy/clientgo_cached_proxy_resolver_test.go

```diff
@@ -134,3 +136,20 @@ func TestCaching(t *testing.T) {
 	proxy.syncTicker.Reset(time.Millisecond * 200)
 	assert.Eventually(t, func() bool { return fakeKongAdminUpdateCount() == previousUpdateCount+1 }, time.Second*10, time.Millisecond*200)
 }
+
+func TestProxyTimeout(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	t.Log("configuring and starting a new proxy server")
+
+	timeout := time.Millisecond * 10
+	fakeKongAdminAPI.Endpoint = httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		mux := http.NewServeMux()
+		mux.HandleFunc("/", func(writer http.ResponseWriter, req *http.Request) {
+			time.Sleep(time.Millisecond * 20) // so we are sure we exceed the timeout
+		})
```

Comment:
Correct me if I'm mistaken, but I don't expect we'll need `mux` here, we can just generically sleep past the timeout since we're testing no other functionality, right?

```suggestion
		time.Sleep(time.Millisecond * 20) // so we are sure we exceed the timeout
```

Reply:
yes , will remove it